macOS/DuckDuckGo/Permissions/View/PopupBlockedPopover.swift

```diff
@@ -39,28 +45,163 @@ final class PopupBlockedPopover: NSPopover {
     }
 
     // swiftlint:disable force_cast
+    var viewController: PopupBlockedViewController {
+        get {
+            if contentViewController == nil {
+                setupContentController()
+            }
+            return contentViewController as! PopupBlockedViewController
+        }
+    }
+    // swiftlint:enable force_cast
+
     private func setupContentController() {
+        let controller: PopupBlockedViewController
+
+        if featureFlagger.isFeatureOn(.newPermissionView) {
+            // Create programmatically for SwiftUI
+            controller = PopupBlockedViewController(newPermissionView: true)
+        } else {
+            // Load from storyboard
+            controller = setupStoryboardController()
```

Comment:
I think this is an overkill for such a simple view: could we just drop the storyboard and put the button behind the feature flag?